components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApisApiServiceImpl.java

```diff
@@ -231,17 +251,23 @@ public Response apisApiIdGraphqlSchemaGet(String apiId, String ifNoneMatch, Stri
                 String errorMessage = "Error while retrieving API : " + apiId;
                 RestApiUtil.handleInternalServerError(errorMessage, e, log);
             }
+        } catch (UserStoreException e) {
+            String errorMessage = "Error while checking availability of tenant " + xWSO2Tenant;
+            RestApiUtil.handleInternalServerError(errorMessage, e, log);
```

Comment:
Looks like this is repeated in many places. Shall we use an exception code based fix. Can do this later if need.
https://github.com/wso2/carbon-apimgt/wiki/Exception-handling-in-REST-APIs

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9609/commits/629fd2c956b66218c6f5b9d125e4b0f25fe735e4